### PR TITLE
AB-32 Precompile assets with gulp

### PR DESCRIPTION
**Pull request by michlask**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@ node_modules
 .nyc_output
 coverage
 .DS_store
-docker-compose.ovverride.yml
\ No newline at end of file
+docker-compose.ovverride.yml
+admin/frontend/assets/styles/
```
Comment (wojtek-krysiak) [inline .gitignore]: I think that those files should be in the repo. After introducing gulp without them there will be no styles

File: admin/backend/routes.js [hunk 2]
```diff
@@ -17,6 +17,34 @@ const ResourcesController = require('./controllers/resources-controller')
 class Routes {
   constructor({ admin }) {
     this._admin = admin
+    this.assetsRoot = `${__dirname}/../frontend/assets/`
+  }
+
+  assets(){
```
Comment (frontkickit) [inline admin/backend/routes.js]: space before curly

File: admin/integrations/hapi.js [hunk 3]
```diff
@@ -8,6 +8,7 @@
  */
 
 const Boom = require('boom')
+const inert = require('inert')
```
Comment (frontkickit) [inline admin/integrations/hapi.js]: [OPT] -> Inert

File: gulpfile.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+const sass = require('gulp-sass');
+const autoprefixer = require('gulp-autoprefixer');
+const babel = require('gulp-babel');
+const uglify = require('gulp-uglify');
+const rename = require('gulp-rename');
+const nodemon = require('gulp-nodemon');
+const concat = require('gulp-concat');
+const path = require('path');
+
+gulp.task('sass', function() {
```
Comment (frontkickit) [inline gulpfile.js]: arrow func

File: admin/integrations/hapi.js [hunk 5]
```diff
@@ -38,6 +41,19 @@ module.exports = {
       })
     })
 
+    await server.register(inert);
+    assets.forEach(asset => {
+      server.route({
+        method: 'GET',
+        path: `${admin.options.rootPath}${asset.path}`,
+        handler: {
+          file: function (request) {
```
Comment (frontkickit) [inline admin/integrations/hapi.js]: file: request => asset.src?

File: gulpfile.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    .pipe(gulp.dest('admin/frontend/assets/scripts'));
+});
+
+gulp.task('nodemon', () => {
+  return nodemon({
+    script: 'example.js'
+  })
+});
+
+gulp.task('watch', function (){
```
Comment (frontkickit) [inline gulpfile.js]: arrow

**Pull request merged**